### ISSUE TITLE
Add NDVI percent of maximum layer

### ISSUE THREAD

**Issue by wmjolly** [alexander-petkov/wfas#38]
On https://www.wfas.net:8443/geoserver I have layer called: current_ndvi_fracmax | wfas:current_ndvi_fracmax configured.  I'd like to make that layer available on the AWS GeoServer

I can set up a custom download directory to make this and other layers available for ingest.  Would that work?

MJ

**Comment by alexander-petkov** [comment]
> On https://www.wfas.net:8443/geoserver I have layer called: current_ndvi_fracmax | wfas:current_ndvi_fracmax configured. I'd like to make that layer available on the AWS GeoServer
> 
> I can set up a custom download directory to make this and other layers available for ingest. Would that work?
> 
> MJ

Maybe simply cascade from wfas,net? I will give it a try.

**Comment by wmjolly** [comment]
No, this one we need to move off of the wfas.net GeoServer completely.  We
will soon be forced to get rid of the Rackspace server and I'm trying to
get everything we need on the AWS GeoServer.

**Comment by alexander-petkov** [comment]
> I can set up a custom download directory to make this and other layers available for ingest. Would that work?

Yes, that will work

**Comment by wmjolly** [comment]
Need to put this back on the list to complete. I'll have a couple of layers that we'll need to add from WFAS to the AWS GeoServer instances. Relative NDVI and Growing Season Index are those.

**Comment by wmjolly** [comment]
I created a download directory on Rackspace and symlink'd the ndvi_perc_max file.  It is here: http://www.wfas.net/data/download/

**Comment by alexander-petkov** [comment]
To add compression and tiling:
```
gdal_translate -f GTiff -co 'COMPRESS=DEFLATE' -co 'TILED=YES' \
   current_ndvi_fracmax.tif current_ndvi_fracmax_deflate.tif 
```
Overviews will be useful too, if this layer will be displayed on a map. They add about 11MB to the file size:

```
gdaladdo current_ndvi_fracmax_deflate.tif 2 4 8 16
```

**Comment by alexander-petkov** [comment]
Available at:
https://aws.wfas.net/geoserver/wcs?service=WCS&version=2.0.1&request=DescribeCoverage&coverageId=wfas:current_ndvi_fracmax
